hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -135,6 +135,8 @@
 
     private String quorumName;
 
+    private boolean hotRestart;
```

Comment:
i think, we also update MapConfig.isCompatible method, since all nodes should have same `hot-restart` config.